README.md

```diff
@@ -124,6 +124,51 @@ message: "app [app_manager/appA] started"
 namespace: "/robot/application"
 ```
 
+## Plugins
+
+You can define `app_manager` plugins as below in app file such as `test.app`.
+
+```yaml
```

Comment:
```suggestion
```yaml
display: Test app
platform: all
launch: test_app_manager/test_app.xml
interface: test_app_manager/test_app.interface
```

It's better to give a full yaml example here. I was wondering why there was no launch file mentioned

Reply:
@v4hn thanks, I updated to add full yaml example.